tests/generate_jwt_keys.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+// if not set, generates just two different keys, both for about 2 hours
+define('JWT_TOKEN_1', getenv('TEST_RABBITMQ_JWT_TOKEN_1') ? getenv('TEST_RABBITMQ_JWT_TOKEN_1') : generateJwtToken(7200));
+define('JWT_TOKEN_2', getenv('TEST_RABBITMQ_JWT_TOKEN_2') ? getenv('TEST_RABBITMQ_JWT_TOKEN_2') : generateJwtToken(7205));
+
+function generateJwtToken($expire = 7200)
```

Comment:
this should be part of test, abstract test class or something "on demand" but not defined as global constats, whats the point?